proxy/src/auth/credentials.rs

```diff
@@ -251,6 +251,7 @@ fn project_name_valid(name: &str) -> bool {
 
 #[cfg(test)]
 mod tests {
+    #![expect(clippy::unwrap_used)]
```

Comment:
nit: Personally, I prefer exterior attributes. If we keep this please add a blank line after them. I wish rustfmt could do this.

Reply:
Actually, there is [`clippy::empty_line_after_outer_attr`](https://rust-lang.github.io/rust-clippy/stable/index.html#/empty_line_after_outer_attr). Should catch this.